input/fsh/SD_Bundle.fsh

```diff
@@ -4,24 +4,23 @@ Id: mcode-patient-bundle
 Title: "mCODE Patient Bundle"
 Description: "A collection of data for an mCODE cancer patient.
 
-The bundle MUST contain exactly one Patient resource conforming to the [CancerPatient](StructureDefinition-mcode-cancer-patient.html) profile, and one or more Condition resource(s) conforming to the [PrimaryCancerCondition](StructureDefinition-mcode-primary-cancer-condition.html) profile.
+The bundle MUST contain exactly one Patient resource conforming to the [CancerPatient](StructureDefinition-mcode-cancer-patient.html) profile.
```

Comment:
Why is `primaryCancerCondition` `0..*` now?

Reply:
I don't think the diagnosis is necessarily available to everyone in the mCODE ecosystem. For example, an #mcode-everything from FoundationOne is a genomics report, and likely nothing else. They care only what's happening at the molecular level. Moreover, we do have words that say, if the information is available, it must be sent. So I think this creates more flexibility and ensures that we don't block reporting information when the PCC is not available.